release-tools/OWNERS

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
```

Comment:
can we get some _local_ names here, like you, Sid, Rob, me, Krish, Sagan?

Reply:
will do, I still need to sanitize lots of things. It does not matter for our repos. I am working on the official repo with the necessary changes.